app/src/test/java/com/vero/tasky/agenda/data/local/dao/ReminderDaoFake.kt

```diff
@@ -20,6 +20,10 @@ class ReminderDaoFake : ReminderDao {
         return  flowOf(reminderList.toList())
     }
 
+    override suspend fun loadReminder(id: String): ReminderEntity {
+        return reminderList.find { it.id == id }!!
```

Comment:
Pretty sure you can use .first instead of find
Which should return a non nullable item

Reply:
And then throws a `NoElementException` if there's no such element in the list, so you again end up with a crash. Doesn't matter overall. In this case, both are absolutely fine since it's just for test cases. This will only break if there's an issue in the test code and in that case it would tell you with a failing test.